tests/support/cluster_util.tcl

```diff
@@ -36,7 +36,7 @@ proc wait_for_cluster_propagation {} {
 
 # Wait for cluster size to be consistent across nodes.
 proc wait_for_cluster_size {cluster_size} {
-    wait_for_condition 50 100 {
+    wait_for_condition 1000 50 {
```

Comment:
Good, if it solves the problem. :+1:

`wait_for_cluster_propagation` uses 5s though (wait_for_condition 50 100), defined just above this function. Is that no problem?

Reply:
i did not see any failures in wait_for_cluster_propagation, so i suppose it is ok.
on the other hands, i am ok with changing it in case the future failure.